packages/frontend/apps/electron/src/main/recording/types.ts

```diff
@@ -0,0 +1,38 @@
+import type { AudioTapStream, TappableApplication } from '@affine/native';
+
+export interface TappableAppInfo {
+  rawInstance: TappableApplication;
+  isRunning: boolean;
+  processId: number;
+  processGroupId: number;
+  bundleIdentifier: string;
+  name: string;
+}
+
+export interface AppGroupInfo {
```

Comment:
what's AppGroup distinguished from App

Reply:
App here actually means a "process".
e.g., Slack app is composed with multiple processes, like `Slack`, `Slack Helper` etc.
The process that are listening to the audio input might be `Slack Helper`.

However the app info like name & icon is only available to the lead process of the app group (mostly be the process that has the app group id).